content/applications/inventory_and_mrp/manufacturing/quality_control/quality_alerts.rst

```diff
@@ -0,0 +1,74 @@
+:menuselection:`Quality Control --> Quality Alerts --> Create`. The quality alert form can then be
+filled out as follows:
+
+- :guilabel:`Title`: choose a concise, yet descriptive title for the quality alert
+- :guilabel:`Product`: the product about which the quality alert is being created
+- :guilabel:`Product Variant`: the specific variant of the product that has the quality issue, if
+  applicable
+- :guilabel:`Lot`: the lot number assigned to the product
+- :guilabel:`Work Center`: the work center where the quality issue originated
+- :guilabel:`Picking`: the picking operation during which the quality issue originated
+- :guilabel:`Team`: the quality team that will be notified by the quality alert
+- :guilabel:`Responsible`: the individual responsible for managing the quality alert
+- :guilabel:`Tags`: classify the quality alert based on user-created tags
+- :guilabel:`Root Cause`: the cause of the quality issue, if known
+- :guilabel:`Priority`: assign a priority between one and three stars to ensure more
+  urgent issues are prioritized
+
+The tabs at the bottom of the form can be used to provide additional information to quality teams:
+
+- :guilabel:`Description`: provide additional details about the quality issue
+- :guilabel:`Corrective Actions`: the method for fixing affected products
+- :guilabel:`Preventive Actions`: procedures for preventing the issue from occurring in
+  the future
+- :guilabel:`Miscellaneous`: the product vendor (if applicable), the company that produces
+  the product, and the date assigned
+
+.. image:: quality_alerts/quality-alert-form.png
+   :align: center
+   :alt: An example of a completed quality alert form.
+
+Create quality alerts during the manufacturing process
+======================================================
+
+Odoo enables manufacturing employees to create quality alerts within a work order without accessing
+the *Quality* module. From the work order tablet view, click the :guilabel:`Menu` button in the top
+left corner and select :guilabel:`Quality Alert`.
+
+.. image:: quality_alerts/work-order-tablet-view-menu-button.png
+   :align: center
+   :alt: Access the work order menu.
+
+The quality alert form can then be filled out as detailed in the previous section. After saving the
+form, a new alert will appear on the :guilabel:`Quality Alerts` page of the *Quality* module.
+
+Manage existing quality alerts
+==============================
+
+By default, quality alerts are organized in a kanban board view. The stages of the kanban board are
+fully configurable and alerts can be moved from one stage to the next by dragging and dropping or
+from within each alert. Additional options are available for viewing alerts, including graph,
+calendar, and pivot table views.
+
+.. tip::
+   Users can filter alerts based on criteria like date assigned or date closed. Alerts can also be
+   grouped by quality team, root cause, or other details.
```

Comment:
Favor active instruction here instead of having "users" as the subject. Added a little bit more specificity too.

The parameters like _Date Assigned_, _Date Closed_, etc.  are borderline `guilabels` since they're choices on the **Filters** menu, but I _think_ it's best to leave them since it's mostly a hypothetical we're entertaining here and not an active instruction. Alternatively we could capitalize as they're reflected on the UI and italicize. Up to you 🙂 
```suggestion
.. tip::
   Filter alerts based on diverse criteria like date assigned or date closed. Alerts can also be
   grouped by quality team, root cause, or other details.
```